projects/go-lib/src/lib/components/go-copy/go-copy.component.scss

```diff
@@ -22,3 +24,7 @@
   overflow: hidden;
   position: absolute;
 }
+
+.go-copy__icon--clicked {
+  color: $base-primary;
```

Comment:
What do you think about using our `$ui-color-positive` variable for this instead?

Reply:
Oh yep, that would make more sense 👍